### PR TITLE
doc fix

### PR DESCRIPTION
**Pull request by lingtangraster**
@hanhu7 Please review, thanks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,117 @@
...
+   git clone https://github.com/Esri/raster-deep-learning.git 
+   ```
+   
+2. Download sample images and sample trained deep learning models (Link is TBD)
+
+3. Set up python deep learning environment in ArcGIS, and install required deep learning libraries in *[requirements.txt](requirements.txt)*.
+   
+   - ArcGIS Pro users can install through *[ArcGIS Pro Python Package Manager](http://pro.arcgis.com/en/pro-app/arcpy/get-started/what-is-conda.htm)*.
+   
+   - Alternatively, ArcGIs Pro users can use this batch file *[env_setup.bat](env_setup.bat)* at root directory to automatically install
```
Comment (hanhu7) [inline README.md]:
```suggestion
   - Alternatively, ArcGIS Pro users can use this batch file *[env_setup.bat](env_setup.bat)* at root directory to automatically install 
```

File: README.md [hunk 2]
```diff
@@ -1,26 +1,117 @@
...
+   
+2. Download sample images and sample trained deep learning models (Link is TBD)
+
+3. Set up python deep learning environment in ArcGIS, and install required deep learning libraries in *[requirements.txt](requirements.txt)*.
+   
+   - ArcGIS Pro users can install through *[ArcGIS Pro Python Package Manager](http://pro.arcgis.com/en/pro-app/arcpy/get-started/what-is-conda.htm)*.
+   
+   - Alternatively, ArcGIs Pro users can use this batch file *[env_setup.bat](env_setup.bat)* at root directory to automatically install 
+   all the required deep learning libraries for the sample cases in this repository into ArcGIS Pro. 
+   Note: use it with caution, in default batch installation will take 5 GB disk space on C: drive. To execute the batch file,
```
Comment (hanhu7) [inline README.md]:
```suggestion
   Note: use it with caution, by default batch installation will take 5 GB disk space on C: drive. To execute the batch file, 
```

File: env_setup.bat [hunk 3]
```diff
@@ -0,0 +1,29 @@
+SETLOCAL
+SET SCRIPTS_DIR=C:\Program Files\ArcGIS\Pro\bin\Python\Scripts\
+SET ENV_DIR=C:\Program Files\ArcGIS\Pro\bin\Python\envs\
```
Comment (hanhu7) [inline env_setup.bat]: Can these two variables dynamically acquired in stead of hard code?

Comment (lingtangraster) [reply env_setup.bat]: added ArcGIS Pro install path acquisition.

File: README.md (hunk 2 above)
Comment (lingtangraster) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (lingtangraster) [reply README.md]: done

**Pull request merged**